portfolio.html

```diff
@@ -0,0 +1,64 @@
+  <link href="css/style.css" rel="stylesheet" type="text/css">
+  <link href="https://fonts.googleapis.com/css?family=Anton|Cinzel|Exo:700|Hind+Siliguri|K2D|Pathway+Gothic+One&display=swap" rel="stylesheet">
+  <title>Portfolio Page</title>
+</head>
+<body>
+  <div class="page-container">
+    <div class="content-wrap">
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="contact.html">Contact</a></li>
+        <li><a class ="active" href="portfolio.html">Portfolio</a></li>
+      </ul>
+      <div class="title-info">
+        <h1>Web Development Projects</h1>
+      </div>
+      <div class="projectDiv">
+        <div class="row">
+          <div class="row-item column">
+            <img src="images/Trains.jpg" alt="attemp1" class="refImage">
+            <a href="https://github.com/buelowek1225"><img src="images/GitHub.png" alt="githublink" class="gitLink"></a>
+            <a href="https://github.com/buelowek1225" class="siteLink">Place Holder</a>
+          </div>
+          <div class="row-item column">
+            <img src="images/LiriBot.jpg" alt="attemp1" class="refImage">
+            <a href="https://github.com/buelowek1225"><img src="images/GitHub.png" alt="githublink" class="gitLink"></a>
+            <a href="https://github.com/buelowek1225" class="siteLink">Place Holder</a>
+          </div>
+          <div class="row-item column">
+            <img src="images/JamFinder.jpg" alt="attemp1" class="refImage">
+            <a href="https://github.com/buelowek1225"><img src="images/GitHub.png" alt="githublink" class="gitLink"></a>
+            <a href="https://github.com/buelowek1225" class="siteLink">Place Holder</a>
+          </div>
+        </div>
+        <div class="row">
+          <div class="row-item column">
+            <img src="images/Google-checkout.jpg" alt="attemp1" class="refImage">
+            <a href="https://github.com/buelowek1225"><img src="images/GitHub.png" alt="githublink" class="gitLink"></a>
+            <a href="https://github.com/buelowek1225" class="siteLink">Place Holder</a>
+          </div>
+          <div class="row-item column">
+              <img src="images/GiddyApp.jpg" alt="attemp1" class="refImage">
+              <a href="https://github.com/buelowek1225"><img src="images/GitHub.png" alt="githublink" class="gitLink"></a>
+              <a href="https://github.com/buelowek1225" class="siteLink">Place Holder</a>
+          </div>
+          <div class="row-item column">
+            <img src="images/Burger.jpg" alt="attemp1" class="refImage">
+            <a href="https://github.com/buelowek1225"><img src="images/GitHub.png" alt="githublink" class="gitLink"></a>
+            <a href="https://github.com/buelowek1225" class="siteLink">Place Holder</a>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="footer">
+      <li>contact: buelowek1225@gmail.com</li>
+    </div>
+  </div>
+</body>
+</html>
```

Comment:
Add a newline to end of the file.

Reply:
Lines changes to include one line at the end of each file.